app/src/main/java/org/dhis2/usescases/main/MainActivity.kt

```diff
@@ -107,24 +110,24 @@ class MainActivity :
 
         binding.filterRecycler.adapter = newAdapter
 
+        binding.navigationBar.pageConfiguration(pageConfigurator)
         binding.navigationBar.setOnNavigationItemSelectedListener {
             when (it.itemId) {
-                R.id.navigation_tasks -> {}
-                R.id.navigation_programs -> {}
-                R.id.navigation_analytics -> {}
-                else -> {}
+                R.id.navigation_tasks -> {
```

Comment:
If is not possible tasks on home, should we have this branch ?

Reply:
The bottomNavigationBar has a placeholder for tasks, so I thought it would be a good idea to leave that piece of code there to implement in the future. But it can be removed sure